test/pretty-print.test.js

```diff
@@ -238,3 +238,176 @@ test('pretty print commonPrefix - handle constrained routes', t => {
   t.is(typeof arrayTree, 'string')
   t.equal(arrayTree, arrayExpected)
 })
+
+test('pretty print includeMeta - commonPrefix: true', t => {
+  t.plan(6)
+
+  const findMyWay = FindMyWay()
+  const namedFunction = () => {}
+  const store = {
+    onRequest: [() => {}, namedFunction],
+    onTimeout: [() => {}],
+    genericMeta: 'meta',
+    mixedMeta: ['mixed items', { an: 'object' }],
+    objectMeta: { one: '1', two: 2 },
+    functionMeta: namedFunction
```

Comment:
Could you add a test with a `Symbol` too?

Reply:
Yup! I added support in the bundling function for symbols as both keys and values (and the appropriate tests)